stories/Button/Guideline.mdx

```diff
@@ -0,0 +1,169 @@
+
+<Story of={ButtonStories.Danger} />
+
+### Links
+
+_to add_
+
+### Size
+
+Only 2 size are available for the button component : default and inline
+
+_Usecases of the different sizes ?_
+
+<Story of={ButtonStories.ButtonSizes} />
+
+### States
+
+#### Disabled button
+
+Disabled state only if an action on the current screen can enable it (action possibility).
+
+<Story of={ButtonStories.ButtonDisabled} />
+
+## Group of Button
+
+- Primary buttons placed on the right of a group.
+- Dismissive actions placed on the left. It allows the user to return to the previous screen or step in the process.
+
+Align button groups with the container's right side by default; however, based on the use case and visual balance it can be aligned differently.
+
+<Story of={ButtonStories.SimpleForm} />
+
+## Icon on buttons
+
+Every button gets a label.\
+Icons are optional and mainly used to show how buttons are different, to aid memory and differentiation.\
+Universally understood icons work well (ie. print, close, play/pause, save).\
+Use standard icons when their use matches their meaning, or at least the user's intent.\
+Avoid creating new icons for every action, especially infrequently used ones.\
+Avoid using an icon alone in a button. If no label is provide, then use a tooltip.
+
+### Placement
+
+When use with a label, the icon is placed on the left side.\
+The icon should be (approximately) the same height as the text.  
+Seeing the icon first help users to scan the page more easily, except for few cases such as navigation (right arrow).
+
+<Story of={ButtonStories.ButtonsWithIcon} />
+
+Avoid using an icon without label, used Universally understood icons such as Save or Delete.
+
+<Canvas of={ButtonStories.IconButtonWithTooltip} />
+
+<Unstyled>
+  <Story of={ButtonStories.IconButtonWithTooltip} />
+</Unstyled>
+
+## Ghost Buttons
+
+Ghost buttons are subtle and contribute to a minimalist interface design.  
+They are low-emphasis, making them ideal for non-primary actions in UIs.  
+They're also effective in space-limited areas like tables or key-value lists.
+
+| Do                                              | Don't                                                                                                        |
+| ----------------------------------------------- | ------------------------------------------------------------------------------------------------------------ |
+| Ensure tooltips are included for clarity        | Confuse them with non-interactive elements like descriptive icons or status indicators                       |
+| Use them for secondary or less crucial actions. | Opt for them over Outline buttons, which are more visibly recognizable as buttons.                           |
+|                                                 | Create too many different instances of ghost buttons; limit them to common actions (e.g. link to other UIs). |
+
+<Story of={ButtonStories.GhostButtons} />
```

Comment:
The buttons should have tooltips on hover, if we want to be consistent with what we described earlier.